Duplicati/Library/Backend/pCloud/pCloudBackend.cs

```diff
@@ -377,15 +387,12 @@ await requestResources.HttpClient.DownloadFile(requestResources.RequestMessage,
     /// <exception cref="Exception">Exceptions arising from either code execution or business logic when return code from pcloud indicates an error.</exception>
     public Task DeleteAsync(string remotename, CancellationToken cancellationToken)
     {
-        if (!String.IsNullOrWhiteSpace(_Path))
-            remotename = $"/{_Path}/{remotename}";
-
         using var timeoutToken = new CancellationTokenSource();
         timeoutToken.CancelAfter(TimeSpan.FromSeconds(SHORT_OPERATION_TIMEOUT_SECONDS));
         using var combinedTokens =
             CancellationTokenSource.CreateLinkedTokenSource(timeoutToken.Token, cancellationToken);
 
-        using var requestResources = CreateRequest($"/deletefile?path={remotename}", HttpMethod.Get);
+        using var requestResources = CreateRequest($"/deletefile?fileid={GetFileId(remotename).Await()}", HttpMethod.Get);
```

Comment:
This should be:
```
using var requestResources = CreateRequest($"/deletefile?fileid={await GetFileId(remotename, cancellationToken).ConfigureAwait(false)}", HttpMethod.Get);
```

Reply:
Latest commit has implemented this alongside with all cancellation token propagation.